test/CompilationDatabase/basic.py

```diff
@@ -0,0 +1,7 @@
+import TestSCons
+
+test = TestSCons.TestSCons()
+test.file_fixture('fixture/SConstruct')
+test.file_fixture('test_main.c')
+test.run()
+test.must_exist('compile_commands.jxson')
```

Comment:
jxson?

Reply:
was just testing that it would fail.. still WIP.. ;)